test.sh

```diff
@@ -35,6 +35,30 @@ curl -sSf -u $SK: $HOST/v1/plans \
    -d currency=eur \
    -d interval=year
 
+curl -sSf -u $SK: $HOST/v1/plans \
+   -H "Content-Type: application/json" \
+   -d '{
+         "id": "annuel-tiered",
+         "currency": "eur",
+         "interval": "year",
+         "interval_count": "1",
+         "usage_type": "licensed",
+         "billing_scheme": "tiered",
+         "tiers_mode": "volume",
+         "tiers": {
+            "0": {
+                  "unit_amount": "0",
+                  "up_to": "1"
+            },
+            "1": {
+                  "flat_amount": "1200",
+                  "unit_amount": "1500",
+                  "up_to": "inf"
+            }
+         },
+         "name": "Annuel tiered"
+   }'
```

Comment:
I wanted to follow the same approach as the other tests here for consistency (using `x-www-formurlencoded` instead of `application/json` for the content type) but it was tricky with tiers being represented as nested objects. @adrienverge happy to change it if you can provide the `x-www-formurlencoded` alternative to this approach